### DISCUSSION TITLE
Alias regex failing

### DISCUSSION THREAD

**Discussion by skumaravelan** [docsifyjs/docsify#1680] [Q&A]
Hi Team, 
I am trying to use alias to resolve the files in other repo. While doing that I need to add  to token to get access. 
/atom-db/README':'https://raw.github.com/atom/atom-db/main/README.md?token=Mytoken

When I try to have the above alias, the value is getting resolved tohttps://raw.github.com/atom-db/main//main/README.md?token= Mytoken.md.

The result is suffixing with".md" . Please help me resolve this issue. 

Senthil

**Comment by trusktr** [comment]
Hello! I don't remember right now what it might be, but maybe useful to check in `router`

**Reply by trusktr** [reply]
Do you have a live example?

**Comment by skumaravelan** [comment]
Sorry, I don't have live example

**Reply by trusktr** [reply]
(you can reply to the same subthread). I think that's just how it works. You may have to fork and customize the router. If you notice there is a bug, let's reproduce it.

**Comment by skumaravelan** [comment]
I am trying to find an example to use router..

**Reply by trusktr** [reply]
There isn't really an example, you'd need to modify the router code (search for "router" in the code base).

**Comment by trusktr** [comment]
Sorry about this issue. I think this can be considered a bug. Mind making a reproduction and putting it in an issue?
